tests/test_workers/test_config.py

```diff
@@ -2,6 +2,7 @@
 from unittest.mock import patch
 from io import BytesIO
 
+import os
```

Comment:
Could you please move this import to the first section since this is a standard library import?